### ISSUE TITLE
The instructions to setup Fission & test function are not working

### ISSUE THREAD

**Issue by adolsalamanca** [fission/fission#2978]
**Fission/Kubernetes version**

<pre>
```
fission check

fission-services
--------------------
√ executor is running fine
√ router is running fine
√ storagesvc is running fine
√ webhook is running fine

fission-version
--------------------
√ fission is up-to-date

kubectl version   
Client Version: v1.29.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.27.10

</pre>

**Kubernetes platform (e.g. Google Kubernetes Engine)**
I'm running in the Cloud via ASG, not managed k8s.

**Describe the bug**
I have just installed fission via helm, as stated in your docs.
After that, I followed the instructions to create both nodejs and python envs, followed by the corresponding hello funcs.

Then I try to test them, and it simply doesn't work.

**To Reproduce**
```
fission function list --namespace fission
NAME  ENV    EXECUTORTYPE MINSCALE MAXSCALE MINCPU MAXCPU MINMEMORY MAXMEMORY SECRETS CONFIGMAPS NAMESPACE
hello python poolmgr      0        0        0      0      0         0                            fission

fission function test --name hello --namespace fission
Error: calling function hello: 404; Please try again or fix the error: 404 page not found

Error: getting function logs: no active pods found for function in namespace fission. Try to get logs from log database.

Options:
  --name=''               Function name
  --method=[GET]          HTTP Methods: GET,POST,PUT,DELETE,HEAD. To mention single method: --method GET
                          and for multiple methods --method GET --method POST. [DEPRECATED for 'fn create',
                          use 'route create' instead]
  --header=[] (-H)        Request headers
  --body='' (-b)          Request body
  --query=[] (-q)         Request query parameters: -q key1=value1 -q key2=value2
  --timeout=1m0s (-t)     Length of time to wait for the response. If set to zero or negative number, no
                          timeout is set
  --dbtype='kubernetes'   Log database type, e.g. influxdb (currently influxdb and kubernetes logs are
                          supported)
  --subpath=''            Sub Path to check if function internally supports routing

Global Options:
  --verbosity=1 (-v)   CLI verbosity (0 is quiet, 1 is the default, 2 is verbose)
  --kube-context=''    Kubernetes context to be used for the execution of Fission commands
  --namespace='' (-n)  If present, the namespace scope for this CLI request

Usage:
  fission function test [options]

Error: error getting function response
```

**Expected result**
I expect being able to run a function.

**Actual result**
It fails, I cannot test fission.

**Screenshots/Dump file**
![image](https://github.com/fission/fission/assets/15842171/d44eeeb2-edbc-412e-a91d-afdd38cc9aa1)


I see it's possible to provide a dump, but I don't believe I have any special setup, I limited my tests to the instructions provided after installing the helm chart, instructions below:

```
NOTES:
1. Install the client CLI.

Mac:
  $ curl -Lo fission https://github.com/fission/fission/releases/download/v1.20.1/fission-v1.20.1-darwin-amd64 && chmod +x fission && sudo mv fission /usr/local/bin/

Linux:
  $ curl -Lo fission https://github.com/fission/fission/releases/download/v1.20.1/fission-v1.20.1-linux-amd64 && chmod +x fission && sudo mv fission /usr/local/bin/

Windows:
  For Windows, you can use the linux binary on WSL. Or you can download this windows executable: https://github.com/fission/fission/releases/download/v1.20.1/fission-v1.20.1-windows-amd64.exe

2. You're ready to use Fission!
  You can create fission resources in the namespace "default"

  # Create an environment
  $ fission env create --name nodejs --image fission/node-env --namespace default

  # Get a hello world
  $ curl https://raw.githubusercontent.com/fission/examples/master/nodejs/hello.js > hello.js

  # Register this function with Fission
  $ fission function create --name hello --env nodejs --code hello.js --namespace default

  # Run this function
  $ fission function test --name hello --namespace default
  Hello, world!
```

I did several tests, noticed that if I install fission in a custom namespace, the fission check command fails, but tried in the default one and it's not working either.

**Comment by soharab-ic** [comment]
Please try below steps:
- Create Fission's resources in default namespace. If you want to create Fission's resources in a custom namespace then you need to set `additionaFissionNamespace` field in helm `values.yaml` file.
```
fission env create --name nodejs --image fission/node-env --namespace default
fission function create --name hello --env nodejs --code hello.js --namespace default
fission function test --name hello --namespace default
```


> if I install fission in a custom namespace, the fission check command fails, but tried in the default one and it's not working either.
- This is fixed in Fission version `v1.20.2`.

**Comment by adolsalamanca** [comment]
It's working well, that was my fault due to assuming default namespace wasn't a requirement in those commands.

Closing as there's no issue.
Thanks!
